Userland/Libraries/LibSQL/AST/Expression.cpp

```diff
@@ -242,9 +242,16 @@ Value MatchExpression::evaluate(ExecutionContext& context) const
         auto result = regex.match(lhs_value.to_string(), PosixFlags::Insensitive | PosixFlags::Unicode);
         return Value(invert_expression() ? !result.success : result.success);
     }
+    case MatchOperator::Regexp: {
+        Value lhs_value = lhs()->evaluate(context);
+        Value rhs_value = rhs()->evaluate(context);
+
+        auto regex = Regex<PosixExtended>(rhs_value.to_string());
```

Comment:
Please check for any errors here, LIKE's regex is generated by us, and cannot contain errors, but this is user-supplied.

Reply:
Addressed by verifying the error status from the regex object, prepending "Regular expression: " in order to clarify the origin of the message and passing it on as an error in the LibSQL context. This means it is presented to the user as decently human-readable error message.